src/Servers/IIS/IIS/test/IIS.Tests/ResponseBodySizeTests.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Server.IIS.FunctionalTests;
+using Microsoft.AspNetCore.Testing;
+
+namespace IIS.Tests;
```

Comment:
@Tratcher is it worth running this test in any other functional permutations like IISExpress.Tests?